pkg/result/filter_test.go

```diff
@@ -685,6 +685,66 @@ func TestFilter(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "ignore file for misconf",
+			args: args{
+				report: types.Report{
+					Results: types.Results{
+						{
+							Misconfigurations: []types.DetectedMisconfiguration{
+								{
+									ID:          "AVD-TEST-0001",
+									AVDID:       "AVD-TEST-0001",
+									Title:       "test-0001",
+									Description: "foo",
+									Severity:    dbTypes.SeverityHigh.String(),
+									Status:      types.StatusFailure,
+								},
+								{ // this misconf is ignored
+									ID:          "AVD-TEST-0002",
+									AVDID:       "AVD-TEST-0002",
+									Title:       "test-0002",
+									Description: "bar",
+									Severity:    dbTypes.SeverityHigh.String(),
+									Status:      types.StatusPassed,
+								},
+								{ // this misconf is ignored
+									ID:          "AVD-TEST-0003",
+									AVDID:       "AVD-TEST-0003",
+									Title:       "test-0003",
+									Description: "baz",
+									Severity:    dbTypes.SeverityHigh.String(),
+									Status:      types.StatusFailure,
+								},
+							},
+						},
+					},
+				},
+				severities: []dbTypes.Severity{dbTypes.SeverityHigh},
+				policyFile: "./testdata/test-ignore-policy-misconf.rego",
+			},
+			want: types.Report{
+				Results: types.Results{
+					{
+						MisconfSummary: &types.MisconfSummary{
+							Successes:  1,
+							Failures:   2,
+							Exceptions: 1,
```

Comment:
Shouldn't it be Failures = 0 and Exceptions = 2 here?

Reply:
Today we show failures that include exceptions. For example:

```shell
Tests: 13 (SUCCESSES: 3, FAILURES: 9, EXCEPTIONS: 1)
Failures: 8 (UNKNOWN: 0, LOW: 2, MEDIUM: 1, HIGH: 5, CRITICAL: 0)
```

In this case, 9 have failed but 1 of them is excepted. The remaining 8 are broken down into the next line by severity.

Reply:
@simar7 Ok, I didn't know that and didn't come across it in the documentation. But still, if two rules are ignored, then shouldn't there be 2 exceptions?

Reply:
1 is a success and 2 are failures. Out of the 2 failures, one is excepted via  the exception rule (ignore policy). Hence 2 failures, 1 success and 1 exception.